### ISSUE TITLE
Out of RAM at scale.

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#90]
More can be done with the logits than with the activations, if the logits look like the real bottleneck. Tomorrow, profile the run at scale, say 5 or 10 percent of the model width, to see what the object at fault is.

Preliminary ideas:
- Sum over positions in the cached activation data, for a factor of 3 memory savings.
- Use just one copy of the base run cached tensor, for a factor of 2 memory savings.
